apps/site/test/site_web/controllers/customer_support_controller_test.exs

```diff
@@ -428,6 +438,44 @@ defmodule SiteWeb.CustomerSupportControllerTest do
     end
   end
 
+  describe "Mode, route and vehicle" do
+    test "get_all_modes" do
+      assert SiteWeb.CustomerSupportController.get_all_modes() == [
```

Comment:
Maybe we should `import SiteWeb.CustomerSupportController` so you don't need to preface all these functions with `SiteWeb.CustomerSupportController`? :)